### ISSUE TITLE
Don't get APP_KEY

### ISSUE THREAD

**Issue by nguyenducvu97** [grokability/snipe-it#15047]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Admin@SGI-IT-VU-L MINGW64 /c/inetpub/wwwroot/snipe-it-7.0.7
$ composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Package operations: 236 installs, 0 updates, 0 removals

In Filesystem.php line 260:

  C:\inetpub\wwwroot\snipe-it-7.0.7/vendor does not exist and could not be cr
  eated:

install [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--dry-run] [--download-only] [--dev] [--no-suggest] [--no-dev] [--no-autoloader] [--no-progress] [--no-install] [--audit] [--audit-format AUDIT-FORMAT] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--] [<packages>...]

![bug](https://github.com/snipe/snipe-it/assets/46434338/27e7273f-a9a4-4674-8ab5-7fb747d652ca)

### Reproduction steps

1.
2.
3.
...

### Expected behavior

composer install

### Snipe-IT Version

v7.0.7

### Operating System

Windows

### Web Server

iis

### PHP Version

MariaDB 11.4

**Comment by snipe** [comment]
You probably need to check your permissions. The vendor directory is created by running composer.

**Comment by nguyenducvu97** [comment]
![image](https://github.com/snipe/snipe-it/assets/46434338/cb15b546-7aa4-4e9a-b49f-eec54306da33)

**Comment by snipe** [comment]
Yeah, I saw that the first time. You probably need to check your permissions. If the web server cannot create the vendors directory, you're gonna have a bad time.
